exercises/throw-an-expression/problem/problem.md

```diff
@@ -0,0 +1,22 @@
+You have been given a piece of code (look for `throw-an-expression.php` in your working directory) which is checking the requested URL and throwing an exception when a secret area of the website is accessed.
+
+If the request is allowed the `Welcome!` is printed out.
+
+Traditionally, pre PHP 8, an exception throw has been a statement. There are certain places where statements can be used, and only expressions can. For example in ternaries and short closures, only expressions can be used.
+
+PHP 8 has changed that, throw statements are now expressions.
```

Comment:
Reading this I feel like it's saying PHP 8 changes where statements can be used.